test/parallel/test-overwrite-config.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+const common = require('../common');
+
+// This test ensures that overwriting a process configuration
+// value does not affect code in bootstrap_node.js.
+
+common.skipIfInspectorDisabled();
+
+const assert = require('assert');
+assert.strictEqual(process.config.variables.v8_enable_inspector, 1);
+process.config.variables.v8_enable_inspector = null;
+assert.strictEqual(process.config.variables.v8_enable_inspector, null);
```

Comment:
How do we know here that the inspector is indeed set up properly (I would have expected to test for inspector specific functions or similar)? That is the most important part out of my perspective. As soon as we know that the inspector is set up, we do not even have to manipulate the variable anymore, because it would not have any effect afterwards.

Reply:
@BridgeAR Thanks for the feedback. I'm not working for a few days but will take another look at this next week.